docs/book/src/tasks/experimental-features/runtime-sdk/implement-extensions.md

```diff
@@ -289,5 +289,39 @@ considerations apply:
 - If there is more than one Runtime Extension registered for the same Runtime Hook and at least one of them fails,
   all the registered Runtime Extension will be retried. See [Idempotence](#idempotence)
 
-Additional considerations about errors that apply only to a specific Runtime Hook will be documented in the hook-specific 
+Additional considerations about errors that apply only to a specific Runtime Hook will be documented in the hook-specific
 implementation documentation.
+
+## Tips & tricks
```

Comment:
What if we add a note about being possible to develop and test with tilt (assuming that github.com/kubernetes-sigs/cluster-api/pull/7154 merge soon, this note could close the circle)

Reply:
Would it fit better into #7154?

Reply:
I think otherwise I have to reference something which isn't documented yet on main or have to wait until after #7154 merges

Reply:
kk, let's make sure this happen in https://github.com/kubernetes-sigs/cluster-api/pull/7154

Reply:
Done > https://github.com/kubernetes-sigs/cluster-api/pull/7154/files#r978283896